Add getAllAWSRoleTypes

rballard-cai opened the pull request:
- Add getAllAWSRoleTypes
- Deprecate getAWSRoleTypes and getNonServiceAWSRoleTypes
- Update jsdoc-to-markdown to version that uses a requizzle version that works with node 12
- Add unit test
- Update docs and dist

pofallon commented:
Thanks for submitting this!  I see where `requizzle` was added but I don't see where it was used?  Do we still need this dev dependency?

americk0 commented on test/test.js:
```diff
@@ -213,6 +213,66 @@ describe('alks.js', function() {
+        _fetch
+      })
+
+      console.log(result)
```
Looks like you missed a console.log while debugging

rballard-cai replied:
Good catch, fixed

rballard-cai commented:
@pofallon it was a nested dependency of jsdoc-to-markdown, I updated the jsdoc-to-markdown version that uses the latest requizzle and deleted the unnecessary requizzle dev dependency.

americk0 commented:
@pofallon when you get the chance could you also approve or request changes? The Github rules on this repo require a second approver
